extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js

```diff
@@ -155,6 +160,43 @@ require(
 			tracker.track(trackingParams);
 		}
 
+		function getWIYLRequestSentKey() {
+			return 'wikiaInYourLangRequestSent' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLNotificationShownKey() {
+			return 'wikiaInYourLangNotificationShown' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLMessageKey() {
+			return targetLanguage + 'WikiaInYourLangMessage' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLLinkTitlesKey() {
+			return targetLanguage + 'WikiaInYourLangLinkTitles' + cacheVersion;
+		}
+
+		function saveLinkTitle(linkAddress) {
+			var linkAddressAry = linkAddress.split('/'),
+				listOfCachedTitles = '',
+				linkTitle = linkAddressAry[linkAddressAry.length - 1].length === 0 ? 'main' : linkAddressAry[linkAddressAry.length - 1];
```

Comment:
I thought "/" should be URL-encoded if it is not a path separator.
So basically, what I am trying to do here is:
If linkAddress = "'http://ja.destiny.wikia.com/'" then, linkAddressAry[linkAddressAry.length - 1] = '' (empty string) so linkTitle = 'main'.
If linkAddress = "http://ja.destiny.wikia.com/wiki/Weapon" then, linkAddressAry[linkAddressAry.length - 1] = 'Weapon', so linkTitle = 'Weapon'. 

Do you mean if there is an URL like "http://ja.destiny.wikia.com/wiki/Foo/Bar", not "http://ja.destiny.wikia.com/wiki/Foo%2FBar"?